tests/test_parsig_api.py

```diff
@@ -95,6 +95,26 @@ def test_parse_sig_latin2(self):
         result = self.sig_parser.parse(sig)[0]
         self.assertEqual(result, expected)
 
+    def test_parse_sig_latin3(self):  # test the latin hour abbreviation q_h (take every _ hours)
```

Comment:
Generally speaking comments are needed only when code does not explain itself. 
take a look in other places at the code to understand the standard